soft4pes/model/grid/rl_grid_param.py

```diff
@@ -0,0 +1,41 @@
+"""
+Parameters for a grid with a stiff voltage source and RL-load.
+"""
+
+import numpy as np
+
+
+class RLGridParameters:
+    """
+    Parameters for a grid with a stiff voltage source and RL-load.
+
+    Parameters
+    ----------
+    Vgr_SI : float
```

Comment:
Has the notation been updated? I find it confusing. Aren't the rated parameters denoted with an uppercase ``R``?

Reply:
This has been removed in the next commit, the grid values can deviate from the rated ones